upgrade/1.3/scripts/sls/README.SLS_Upgrade.md

```diff
@@ -77,7 +77,7 @@ During the update of SLS, at a minimum, answers to the following questions must
 For help and all options, run the following:
 
 ```bash
-ncn# ./sls_updater_csm_1.2.py --help
+ncn# ./sls_updater_csm_1.3.py --help
```

Comment:
The actual script file is not being renamed with this PR, so this command won't exist